src/Shared/Recruit.Vacancies.Client/Infrastructure/Reports/ReportService.cs

```diff
@@ -93,7 +94,7 @@ public void WriteReportAsCsv(Stream stream, Report report)
 
             var results = JArray.Parse(report.Data);
 
-            _csvBuilder.WriteCsvToStream(stream, results, report.RequestedOn.ToUkTime(), reportStrategy.ResolveFormat);
+            _csvBuilder.WriteCsvToStream(stream, results, report.RequestedOn.ToLocalTime(), reportStrategy.ResolveFormat);
```

Comment:
Can you change this back please.  ToLocalTime is no longer used and ToUkTime should now be used.
The App Services run in UTC so Local Time is meaningless.
See this PR for info.  https://github.com/SkillsFundingAgency/das-recruit/pull/621